api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -355,11 +355,12 @@ message JwtProvider {
   // Specify the claim name you want to copy in which HTTP header. For examples, following config:
   // The claim must be of type; string, int, double, bool. Array type claims are not supported
   //
-  // .. code-block:: yaml
-  //
-  //   claim_to_headers:
-  //     - name: x-jwt-claim-nested-claim
-  //       claim: claim.nested.key
+  // .. literalinclude:: /_configs/repo/jwt_authn.yaml
+  //    :language: yaml
+  //    :lines: 44-48
+  //    :linenos
```

Comment:
```suggestion
  //    :linenos:
```

apologies - seems it needs an ending colon

also there is a format issue which should be fixed with:

```diff

diff --git a/tools/spelling/check_spelling_pedantic.py b/tools/spelling/check_spelling_pedantic.py
index 2ea0a70..082122d 100755
--- a/tools/spelling/check_spelling_pedantic.py
+++ b/tools/spelling/check_spelling_pedantic.py
@@ -690,7 +690,8 @@ def extract_comments(lines):
                     break
 
                 n += 1
-        elif text.strip().startswith(RST_CODE_BLOCK) or text.strip().startswith(RST_LITERAL_INCLUDE):
+        elif text.strip().startswith(RST_CODE_BLOCK) or text.strip().startswith(
+                RST_LITERAL_INCLUDE):
             # Start of a code block.
             indent = len(INDENT.search(text).group(1))
             last_line = comments[n].line
```